.gitignore

```diff
@@ -3,4 +3,4 @@ node_modules/
 .coveralls.yml
 .nyc_output/
 coverage/
-.coveralls.yml
\ No newline at end of file
+dist/
```

Comment:
Great Work, I think you should consider EOLs here

Reply:
What do you mean by that?

Reply:
End of Line